lib/usage.js

```diff
@@ -261,10 +261,24 @@ module.exports = function (yargs, y18n) {
       })
 
       examples.forEach(function (example) {
-        ui.div(
-          {text: example[0], padding: [0, 2, 0, 2], width: maxWidth(examples, theWrap) + 4},
-          example[1]
-        )
+        if (example[1] === '') {
+          ui.div(
+            {
+              text: example[0],
+              padding: [0, 2, 0, 2]
```

Comment:
@bcoe I've looked at this again but it seems that even without the width property it takes up the full line:
<img width="388" alt="bildschirmfoto 2017-01-10 um 14 48 51" src="https://cloud.githubusercontent.com/assets/7421183/21808520/f92c4942-d743-11e6-9608-c8fe9c1d5e2e.png">

Isn't that what you meant with `but it will take up a full line`?
Using the width property like you suggested makes it wrap after each character:
<img width="424" alt="bildschirmfoto 2017-01-10 um 14 44 41" src="https://cloud.githubusercontent.com/assets/7421183/21808407/8dc6bb2e-d743-11e6-9462-1e666eb7f089.png">

Anyway, what I besides would suggest is perhaps adding a newline after each example to make it more distinguishable:
<img width="373" alt="bildschirmfoto 2017-01-10 um 14 50 53" src="https://cloud.githubusercontent.com/assets/7421183/21808591/36468acc-d744-11e6-9e6e-4bc28471adb9.png">

This can be achieved very easily with a bottom padding of 1.